README.md

```diff
@@ -82,7 +82,7 @@ We recommend only syncing to specific release tags to make sure you're at a know
 
 You can use CocoaPods to remain up to date with ADAL within a specific major version. Include the following line in your podfile:
 
-    pod 'ADAL', '~> 2.7', :submodules => true
+    pod 'ADAL', '~> 4.0', :submodules => true
```

Comment:
Can you also remove :submodules true from here? this version has been already released.